src/main/java/it/eng/idsa/businesslogic/service/impl/RejectionMessageServiceImpl.java

```diff
@@ -36,25 +35,21 @@ public class RejectionMessageServiceImpl implements RejectionMessageService{
 	
 	private static final Logger logger = LoggerFactory.getLogger(RejectionMessageServiceImpl.class);
 	
-	private DapsTokenProviderService dapsProvider;
-	
 	private SelfDescriptionConfiguration selfDescriptionConfiguration;
 	
-	public RejectionMessageServiceImpl(DapsTokenProviderService dapsProvider,
-			SelfDescriptionConfiguration selfDescriptionConfiguration) {
+	public RejectionMessageServiceImpl(SelfDescriptionConfiguration selfDescriptionConfiguration) {
 		super();
-		this.dapsProvider = dapsProvider;
 		this.selfDescriptionConfiguration = selfDescriptionConfiguration;
 	}
 
 	@Override
-	public void sendRejectionMessage(RejectionMessageType rejectionMessageType, Message message) {
+	public void sendRejectionMessage(Message requestMessage, RejectionMessageType rejectionMessageType) {
```

Comment:
Why changing places for those 2 parameters?
Not important change, but affects a lot of places where this method is called

Reply:
Because of previous changes to the code where I first removed the message parameter from the method. After agreeing with you on how  to proceed next it was easier for me to do it this way.